src/mail/mjml/invitation.mjml

```diff
@@ -3,19 +3,14 @@
 
   <mj-body mj-class="bg--blue-100">
     <mj-wrapper css-class="wrapper" padding="0 25px 0px 25px">
-      <mj-section
-        background-url="{{domain}}/assets/mail-header-background.png"
-        background-size="cover"
-        background-repeat="no-repeat"
-        background-position="0 -30px"
-      >
+      <mj-section>
         <mj-column>
           <mj-image
             align="center"
-            src="{{domain}}/assets/logo-suite-numerique.png"
+            src="{{logo_img}}"
             width="250px"
             align="left"
-            alt="{%trans 'La Suite Numérique' %}"
+            alt="{%trans 'Logo email' %}"
```

Comment:
I think you should now test the presence of the brand and logo in the email sent.
You should be able to do it in this existing test `test_api_document_invitations_create_privileged_members`